R/check-class.R

```diff
@@ -68,6 +68,9 @@ rhub_check <- R6Class(
     print = function(...)
       check_print(self, private, ...),
 
+    browse = function(which = NULL)
+      check_web(self, private, which),
+    
     web = function(which = NULL)
       check_web(self, private, which),
 
```

Comment:
Can `browse` call `web`?

Reply:
How? I tried

```r
browse = function(which = NULL)
      web(which)
```

but then I get the error `Error in web(which) : could not find function "web"` when I try using the `browse` method.

Reply:
maybe I can just make it `self$web(which)` :woman_facepalming:

Reply:
`self$web(which)`